inc/lib/webm/posthandler.php

```diff
@@ -1,27 +1,22 @@
 <?php
 // Glue code for handling a Tinyboard post.
 // Portions of this file are derived from Tinyboard code.
-
 function postHandler($post) {
     global $board, $config;
-
-    if ($post->has_file) foreach ($post->files as &$file) if ($file->extension == 'webm') {
+    if ($post->has_file) foreach ($post->files as &$file) if ($file->extension == 'webm' || $file->extension == 'mp4') {
       if ($config['webm']['use_ffmpeg']) {
         require_once dirname(__FILE__) . '/ffmpeg.php';
         $webminfo = get_webm_info($file->file_path);
-
         if (empty($webminfo['error'])) {
           $file->width = $webminfo['width'];
           $file->height = $webminfo['height'];
-
           if ($config['spoiler_images'] && isset($_POST['spoiler'])) {
             $file = webm_set_spoiler($file);
           }
           else {
             $file = set_thumbnail_dimensions($post, $file);
             $tn_path = $board['dir'] . $config['dir']['thumb'] . $file->file_id . '.jpg';
-
-            if(false == make_webm_thumbnail($file->file_path, $tn_path, $file->thumbwidth, $file->thumbheight)) {
+            if(0 == make_webm_thumbnail($file->file_path, $tn_path, $file->thumbwidth, $file->thumbheight, $webminfo['duration'])) {
```

Comment:
why 0? i understand, that php has really weird type comparison. eg. 0 == "orange". shouldn't it be a stricter check?